### DISCUSSION TITLE
No data for listener metric - gha_job_queue_duration_seconds_sum

### DISCUSSION THREAD

**Discussion by rekha-prakash-maersk** [actions/actions-runner-controller#3378] [Questions]
Hello,

In the listener all metrics are showing up fine except `gha_job_queue_duration_seconds_sum` which is included here - https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/deploying-runner-scale-sets-with-actions-runner-controller#enabling-metrics

This metric crucial to understand the self runner performance, so can anyone please help how to fix this ?

**Comment by rekha-prakash-maersk** [comment]
Hi @nikola-jokic , The metric is commented out here - https://github.com/actions/actions-runner-controller/blob/master/cmd/githubrunnerscalesetlistener/metrics.go#L163-L171, but the doc says its available. Is this supposed to be available ?

**Reply by nikola-jokic** [reply]
Oh, it shouldn't for now. I'll need to fix the docs, sorry. Thank you for noticing it.

**Reply by rekha-prakash-maersk** [reply]
Hi @nikola-jokic okay, Thanks for the update.  Will this metrics be included in any near future releases ?

**Reply by nikola-jokic** [reply]
Unfortunately, I cannot promise when, but it should be included in one of our next releases
